aten/src/ATen/CMakeLists.txt

```diff
@@ -262,31 +284,49 @@ set(TBB_BUILD_TESTS OFF CACHE BOOL " " FORCE)
 add_subdirectory(cpu/tbb)
 set_property(TARGET tbb_static tbb_def_files PROPERTY FOLDER "dependencies")
 target_include_directories(tbb_static PUBLIC ${TBB_ROOT_DIR}/include)
-target_link_libraries(ATen tbb_static)
+target_link_libraries(ATen_cpu tbb_static)
 
 if(NOT ${CMAKE_VERSION} VERSION_LESS "3.1")
-    SET_PROPERTY(TARGET ATen PROPERTY CXX_STANDARD 11)
-endif(NOT ${CMAKE_VERSION} VERSION_LESS "3.1")
+  set_property(TARGET ATen_cpu PROPERTY CXX_STANDARD 11)
+  if(CUDA_FOUND)
+    set_property(TARGET ATen_cuda PROPERTY CXX_STANDARD 11)
```

Comment:
Shouldn't this be guarded by `NOT NO_CUDA` as well?

Reply:
I always forget which one I'm supposed to test for. I think the CMakeLists.txt is also inconsistent about it. Will fix.